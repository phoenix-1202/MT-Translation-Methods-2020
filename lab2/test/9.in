fun letsMakeTheMistakeInOrderOfTokens)a: Int, b: Int): Unit