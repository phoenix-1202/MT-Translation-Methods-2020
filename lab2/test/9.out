Assertion error: expected (, found ) at pos 39